test/DynamoCoreTests/ResourcePathLengthTests.cs

```diff
@@ -0,0 +1,236 @@
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace Dynamo.Tests
+{
+    /// <summary>
+    /// Tests to validate that resource file paths don't exceed Windows path length limitations.
+    /// This addresses the issue where CI servers with long path prefixes cause MSB3103 errors
+    /// when resource paths become too long.
+    /// </summary>
+    [TestFixture]
+    public class ResourcePathLengthTests : UnitTestBase
```

Comment:
@saintentropy I created a new test class here, thinking that we may want to reuse the `resx` transverse logic for further work, as per your original intent to do a follow-up job on the image resource analysis.